console/src/components/Services/Data/TableInsertItem/InsertActions.js

```diff
@@ -40,8 +40,7 @@ const insertItem = (tableName, colValues) => {
       const colSchema = columns.find(x => x.column_name === colName);
       const colType = colSchema.data_type;
       if (Integers.indexOf(colType) > 0) {
-        insertObject[colName] =
-          parseInt(colValues[colName], 10) || colValues[colName];
```

Comment:
The flag `HASURA_GRAPHQL_STRINGIFY_NUMERIC_TYPES` will take care of the response but while sending we may need to avoid this parse.